tests/integration/python/aim/custom.patch

```diff
@@ -1,268 +1,201 @@
 diff --git a/tests/api/test_dashboards_api.py b/tests/api/test_dashboards_api.py
-index 94bc29c..3a8237d 100644
+index 94bc29c..85000fc 100644
```

Comment:
This "diff of a diff" isn't very easy to review -- should we think about an approach like the UI modules for these tests?

Reply:
Agreed that it makes it hard to review. I do wonder if the module approach may be overkill for these tests as they shouldn't change much?